### DISCUSSION TITLE
Does Wix Toolset 4 supports jpackage?

### DISCUSSION THREAD

**Discussion by sblantipodi** [wixtoolset/issues#7982] [Questions]
As title.
I'm using wixtoolset 3 along with jpackage but I cannot make it work with wixtoolset 4.

Does V4 supports jpackage?

**Comment by robmen** [comment]
You have it backward. Nothing in any version of the WiX Toolset depends on JPackage.

**Reply by sblantipodi** [reply]
thanks for the answer.

jpackage is a pretty big player here,
don't you test if you break something with it?

from your answer it seems that you would say: 
"no, we don't care about jpackage".

**Reply by robmen** [reply]
I care deeply about my customers. jpackage is not a customer (as of today).

**Comment by chrpai** [comment]
@sblantipodi Oracle makes a tool that uses WiX and now WiX is responsible for it?  Does Oracle have any money for that support?  WiX v3 still exists so I don't see how anything is broken.

Personally Windows Installer and Windows Installer XML is the anything packager.  Jpackage is a niche tool.  As such I don't have any interest in it.
